package.json

```diff
@@ -22,6 +34,7 @@
     "@babel/cli": "^7.1.2",
     "@babel/preset-env": "^7.1.0",
     "@babel/preset-react": "^7.0.0",
+    "babel-core": "7.0.0-bridge.0",
```

Comment:
Why include `@babel/core` and `babel-core`?

Reply:
its needed.
https://jestjs.io/docs/en/getting-started#using-babel